### DISCUSSION TITLE
Implement disabling a embedding of media

### DISCUSSION THREAD

**Discussion by vitonsky** [umputun/remark42#1386] [Q&A]
When i test the remark42, i have enable anonymous access, and as guest i tried to post. I discovered that as guest i can do something like this:

```md
![](urlToExternalPic.jpg)
```

And picture will shown in comments.

Can i disable pictures? Maybe we should implement markdown AST filtration on backend to solve this problem and allow to users implement other rules to handle comments

**Comment by umputun** [comment]
as of now, anonymous access allows external images but restricts image upload.

**Reply by vitonsky** [reply]
I want to disable media for all users which not approved by me

**Reply by umputun** [reply]
the conditional limiting of some content is doable, and I kind of remember filtering it in some cases. Don't recall where exactly, but sounds familiar. You can check the code, this is likely done with some regex or goquery

**Reply by paskal** [reply]
We have an issue https://github.com/umputun/remark42/issues/1300 about comments pre-moderation support which is not targeting this particular problem but might solve it as well because you would have to approve each new comment to the site before anyone, but the comments moderator sees it. However, I would expect the implementation to take at least two-three months from now as I assume I'm the only person who is interested in writing that functionality.
